demo/c/cobra_demo_file.c

```diff
@@ -32,11 +33,11 @@ static void *open_dl(const char *dl_path) {
 
 #if defined(_WIN32) || defined(_WIN64)
 
-    return LoadLibrary(dl_path);
```

Comment:
I think that 4-space indentation is more common for C. Could you tell `clang-format` to consider this?